access_card/src/main/java/edu/artic/accesscard/AccessMemberCardFragment.kt

```diff
@@ -167,6 +167,10 @@ class AccessMemberCardFragment : BaseViewModelFragment<AccessMemberCardViewModel
                 .map { it.size > 1 }
                 .bindToMain(switchCardHolder.visibility())
                 .disposedBy(disposeBag)
+
+        viewModel.isReciprocalMemberLevel
+                .bindToMain(reciprocalMember.visibility())
```

Comment:
What do you think of assigning the `memberInfo.memberLevel` as contentDescription?

Reply:
Done.